pkg/ddevapp/drupal/drupal10/settings.ddev.php

```diff
@@ -38,7 +38,8 @@
   $settings['config_sync_directory'] = 'sites/default/files/sync';
 }
 
-// Override drupal/symfony_mailer default config to use Mailpit.
+// Override drupal/symfony_mailer default config to use Mailpit
```

Comment:
```suggestion
// Override drupal/symfony_mailer default config to use Mailpit.
```

Fixing phpcs that was fixed in the conflicting PR.